cpp/include/raft/matrix/detail/linewise_op.cuh

```diff
@@ -796,7 +796,7 @@ struct MatrixLinewiseOp {
                   "layout for in and out must be either padded row or col major");
 
     // also statically assert padded matrix alignment == 2^i*VecBytes
-    assert(raft::Pow2<VecBytes>::areSameAlignOffsets(in, out));
+    assert(raft::Pow2<VecBytes>::areSameAlignOffsets(in.data_handle(), out.data_handle()));
```

Comment:
Normally, we use `RAFT_EXPECTS` instead of the plain `assert`. Seems like this one slipped through unnoticed. While you're at it, could you please fix this as well?

Reply:
Done